src/main/java/learn/field_agent/controllers/GlobalExceptionHandler.java

```diff
@@ -33,16 +35,12 @@
         return ErrorResponse.build(result);
     }
 
-    @ExceptionHandler(RuntimeException.class)
-    public ResponseEntity<ErrorResponse> handleException(RuntimeException ex) {
-        Result result = new Result();
-        result.addMessage("Sorry but your request returned no results.", ResultType.NOT_FOUND);
-        return ErrorResponse.build(result);
-    }
-
-
     @ExceptionHandler(Exception.class)
-    public ResponseEntity<ErrorResponse> handleException(Exception ex) {
+    public ResponseEntity<ErrorResponse> handleException(Exception ex) throws Exception {
+        if (ex instanceof HttpMessageNotReadableException || ex instanceof HttpMediaTypeNotSupportedException) {
+            throw ex; // return
+        }
```

Comment:
Rethrowing these exception types so Spring MVC can handle them. This gets two controller tests passing again.